### ISSUE TITLE
increment.me (Anonymous Feedback)

### ISSUE THREAD

**Issue by nelsonic** [dwyl/feedback#97]
This was on the front page of HN this morning: https://news.ycombinator.com/item?id=25029904

https://www.increment.me
![image](https://user-images.githubusercontent.com/194400/98529017-02e4d000-2275-11eb-8d95-1e6e54f588b1.png)

Looks they are still in Beta.

**Comment by iteles** [comment]
Disappointingly (but predictably) closed source.

**Comment by nelsonic** [comment]
Yeah, closed source is the default for most Micro-SaaS, especially when in Beta. 🙄 
Presumably they thought that the idea was so simple that if they made open nobody would pay for it. 💭 
I think the _opposite_ is true. If they Open Source it and get more contributors and build better features faster,
they can get to decent adoption+revenue sooner and focus on experience/branding.
